src/clients/third-party-service-client.ts

```diff
@@ -0,0 +1,61 @@
+import { FiscalCode } from "generated/io-api/FiscalCode";
+import nodeFetch from "node-fetch";
+import {
+  ApiKeyAuthenticationConfig,
+  ThirdPartyConfig
+} from "src/utils/thirdPartyConfig";
+import {
+  Client,
+  createClient
+} from "../../generated/third-party-service/client";
+
+/**
+ *
+ * @param apiKey
+ * @returns
+ */
+const withApiKey = (apiKey: ApiKeyAuthenticationConfig) => (
+  fetchApi: typeof fetch = (nodeFetch as unknown) as typeof fetch // TODO: customize fetch with timeout
```

Comment:
**NB:** the timeout could be set outside using the AbortableFetch (eg config.ts -> getHttpsApiFetchWithBearer)

Reply:
Sorry, I just copy-paste a cast without removing the comments :D